test/components/EditableCard.test.js

```diff
@@ -0,0 +1,66 @@
+/* global it, describe, expect */
+/* eslint-disable sort-keys */
+
+import React from 'react';
+import faker from 'faker';
+import { Tester } from '@mighty-justice/tester';
+
+import { EditableCard } from '../../src';
+
+function changeInput (component, value) {
+  component.simulate('focus');
+  component.simulate('change', { target: { value } });
+  component.simulate('blur');
+}
+
+function sleep (ms = 0) {
```

Comment:
No need, `await tester.sleep();`

Reply:
Have you considered including other utilities like `changeInput` in tester or importable alongside?